docs/technical-documentation/migration-overview.md

```diff
@@ -13,20 +13,20 @@ The three options are:
 
 Why use the rest API?
 - Works anywhere
-  - You don’t have to work on the Drupal server. Migrate from your laptop!
+    - You don’t have to work on the Drupal server. Migrate from your laptop!
```

Comment:
The indentation is working everywhere else, but these three bullets aren't being processes right when I build the docs:

![image](https://user-images.githubusercontent.com/2371345/93102367-cbe1ac00-f681-11ea-9867-924a0f1cacdc.png)

Reply:
😅

Reply:
d'oh! fixed now, with a bit of reformatting.

Reply:
All good now!